cordapp-cpk2/README.md

```diff
@@ -69,13 +69,15 @@ cordapp {
         versionId = 1
         licence = 'Test-Licence'
         vendor = 'R3'
+        cordappName = 'simple.cordapp'
```

Comment:
Should be adding a few words to mention what `cordappName` is for?  I believe it will raise confusion with `name`.